### DISCUSSION TITLE
How can I update the "sourceLayerID" in Qfield Cloud?

### DISCUSSION THREAD

**Discussion by kcfol** [opengisch/QField#3637] [Q&A]
Hi, 
I have a series of points that have been successfully collected and pushed to the qfield cloud from a tablet remotely, but they won't sync to the local qfield project. The "sourceLayerID" is different between the points on the cloud and the "sourceLayerID" in the local workspace. Is there a way to update the "sourceLayerId" of the points on the cloud so that I can sync the data points?

I hope that makes sense. 
Thanks.

**Comment by miili** [comment]
See also issue here https://github.com/opengisch/qfieldcloud/issues/415
